developers/ide/eclipse.md

```diff
@@ -5,65 +5,153 @@ title: Eclipse IDE
 
 # Eclipse IDE
 
-::: tip TODO
-We are currently reworking how to setup a perfect development environment.
-A new step by step guide will appear here soon.
-:::
+Eclipse is the development environment used since the inception of openHAB.
+To make development easier an out-of-the-box setup is available that completely configures Eclipse to easily develop for the openHAB projects.
+This guide describes the steps to setup Eclipse and how to debug an addon in Eclipse.
 
-For now follow these steps:
+## Eclipse IDE Setup
 
-### Eclipse IDE Setup
-
-This guide focuses on existing Add-on (binding) development.
+1. Install the JAVA 8 JDK if you didn't have it installed. Make sure to install **JAVA JDK version 8!**
 
 1. Download the "Eclipse Installer": [can be downloaded from the Eclipse web site](https://wiki.eclipse.org/Eclipse_Installer)
 
-1. Launch the Installer and on the menu on the top right (3 bars) select `ADVANCED MODE...`, then select `Eclipse IDE for Java Developers`.
-Ensure that you select the Eclipse version to use to "2019-03" as version `2019-06` currently does not support bndtools. Finally select `Next >`.
+1. Launch the Installer and on the menu on the top right (3 bars)
+
+    ![installer](./images/ide_setup_eclipse_1_installer.png)
+
+1. Select `ADVANCED MODE...`
+
+    ![installer](./images/ide_setup_eclipse_2_advanced.png)
+
+1. Select `Eclipse IDE for Java Developers`. Select `Next >`.
+
+    ![select](./images/ide_setup_eclipse_3_select_ide.png)
+
+1. Under `Github Projects > openHAB` select `openHAB Development` and any desired option from `openHAB Add-ons` (includes all addons from openhab2-addons repo), `openHAB ZigBee Binding` or `openHAB Z-Wave Binding`.
+
+   ![select projects](./images/ide_setup_eclipse_4_openhab.png)
+
+   |Selection              | Install if
+   |-----------------------|--------------------------------------
+   |**openHAB Development**| **Debug/Demo Environment (Required)**
+   |openHAB Add-ons        | Installs all openHAB 2 Add-ons
+   |openHAB ZigBee Binding | ZigBee Binding Development
+   |openHAB Z-Wave Binding | Z-Wave Binding Development
+   |openHAB BACNet Binding | BACNet Binding Development
+   |openHAB Web UIs        | Web UIs Development
+   |openHAB Core Framework| Core Framework Development
+
+    **NOTE: If you select `openHAB Add-ons` the openHAB2-addons repository will be checked out and all addon projects are imported in Eclipse. If you want to develop a new binding or a specific binding it is recommended to clone your own fork of openHAB2-addons and import only the projects you work on.**
+
+    **If you have selected `openHAB Add-ons` during the installation and these projects have been imported during the setup process and show up in the Package Explorer, select `OH2 Add-ons` and from right-click menu select "Close Projects": *this significantly speeds up the setup*.
+    Re-open only the binding project(s) you would like to work on.**
```

Comment:
Maybe **custom vuepress containers** would make this more visible.
https://v1.vuepress.vuejs.org/guide/markdown.html#custom-containers

Wdyt?